src/mpyl/project.py

```diff
@@ -565,7 +565,7 @@ def validate_project(file: TextIO) -> dict:
     :return: the validated schema
     :raises `jsonschema.exceptions.ValidationError` when validation fails
     """
-    yaml_values = YAML(typ="unsafe").load(file)
+    yaml_values = YAML().load(file)
```

Comment:
This will slow matters down way too much. I would suggest to reload the project (safely) here only https://github.com/Vandebron/mpyl/blob/caf442e2df0d8be92ba33a652fd1e785a6839e7d/src/mpyl/build.py#L155

Because it's only in the context of building, deployment in particular that the yaml parsing is relevant.

Reply:
@ptab I made the safe load conditional here: https://github.com/Vandebron/mpyl/pull/283/commits/b9b3b94baa7f714c371be8d1d65fa3f9df613906

That should fix the issue, but routine operations like `mpyl build status` snappy.

Reply:
Sounds like a good compromise, thanks for finding an alternative solution 👍